### DISCUSSION TITLE
Hello, i have a problem. I can not download a track that exist .  I can't get the gpx because osmand said destination is not on. Osmand give me the coordination?? What can i do

### DISCUSSION THREAD

**Discussion by Lieve2810** [osmandapp/OsmAnd#18799] [General]
Hello, my problem in osmand is that i not get a gpx of the track. He said destination is not on. He gives me coordination..
.What can i do...Before it was always possible. Greetings lieve.
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/153303892/766cd6dc-b05e-4fa9-ba7e-f7bab5c55d77"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/153303892/6e2cb2a4-76f7-4eb8-afb9-b71acef219c6"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by DmitryAlexei** [comment]
What is the version of the app?
Does restarting OsmAnd help to remove this issue?

Can you try to zoom in and tap on the route more precisely?

**Comment by Lieve2810** [comment]
My version is 4.6.9.

I restarted already many times.
The problem is not solved.

Greets
Lieve

Op zo 24 dec. 2023 18:51 schreef DmitryAlexei ***@***.***>:

**Reply by DmitryAlexei** [reply]
Please provide your profile settings (menu > settings > walking profile > export), so we can try to reproduce this issue
